NEO-TAT/tat_flutter
# Debug page for workmanager

## Description 
- adding a debug page for testing the background tasks using `flutter_workmanager` package.

## How to Verify? 
- Go to BGserviceDebugPage and trigger all background tasks to see if they work fine.

## Screenshots/GIF/Test Results (Optional) 
<img src="https://user-images.githubusercontent.com/47718989/168683417-83777e6b-04cb-4c0b-a082-5d8d1e94a1f0.jpg" height="300">  <img src="https://user-images.githubusercontent.com/47718989/168683422-c0bc246d-a7d1-4c14-9a56-f572dcb061b6.jpg" height="300">
